torch/distributed/_tools/memory_tracker.py

```diff
@@ -105,8 +105,10 @@ def start_monitor(self, root_module: nn.Module) -> None:
             # the memory stats tracked here may not completely accurate.
             h1 = m.register_forward_pre_hook(self._create_pre_forward_hook(name))
             h2 = m.register_forward_hook(self._create_post_forward_hook(name))
-            h3 = m.register_backward_hook(self._create_backward_hook(name))
-            self._hooks.extend([h1, h2, h3])
+            # it does not work well with jagged tensor somehow, the root cause is not 
```

Comment:
nit: Should we prefix with `TODO:` to make this easier to search?

Also, lint error looks real.